notebooks/Issue322/322_Influence-surface-thickness_Points.csv

```diff
@@ -0,0 +1,117 @@
+X,Y,Z,surface

```

Comment:
specially for the test, it would be better if you use one of the models that are already there so we do need add more data